.circleci/config.yml

```diff
@@ -107,9 +107,9 @@ workflows:
 
       - architect/push-to-app-collection:
```

Comment:
Can we delete this instead? We don't need release-operator for pure CAPI clusters.

Reply:
sure, i was just doing an automated sed/gh pr create for every repo which pushes to the vsphere collection. happy to remove instead though.